apps/dotcom/sync-worker/src/TLDrawDurableObject.ts

```diff
@@ -935,3 +996,38 @@ async function listAllObjectKeys(bucket: R2Bucket, prefix: string): Promise<stri
 
 	return keys
 }
+
+function* generateSnapshotChunks(snapshot: RoomSnapshot): Generator<Uint8Array> {
```

Comment:
might be worth adding some tests for this? could check that serializing and then parsing again produces the same result as the input snapshot?